src/views/Home/Home.test.jsx

```diff
@@ -11,5 +14,47 @@ const user = {
 }
 
 test('Should render the user profile', () => {
+  render(
+    <MemoryRouter>
+      <Home user={user} />
+    </MemoryRouter>
+  );
 
+  // NAME
+  screen.getByText(/vonta/i);
+  // Motto
+  screen.getByText(/res non verba/i);
+  // Interests
+  screen.getByText(/interests/i);
+  // Avatar
+  screen.getByAltText(/avatar/i);
+  // Header
+  screen.getByAltText(/header/i);
```

Comment:
I would set all of these elements you are finding on the page to a variable, and then you can `expect(variableName).toBeInTheDocument()` or a myriad of other `expect` statements.  These make your test more readable by explicitly stating what you are evaluating for.

Reply:
That makes sense. I'll make sure to do this in the from scratch.